lib/client/mfa/cli.go

```diff
@@ -101,40 +107,71 @@ func (c *CLIPrompt) Run(ctx context.Context, chal *proto.MFAAuthenticateChalleng
 
 	promptOTP := chal.TOTP != nil
 	promptWebauthn := chal.WebauthnChallenge != nil
+	promptSSO := false // TODO(Joerger): check for SSO challenge once added in separate PR.
```

Comment:
I do appreciate the small change but it looks like we are mainly adding dead code to this file. Would it make sense to do a bit more here, so at least we can test the "SSO not implemented" error? Or maybe move some changes to a PR where they are effective?